### PR TITLE
update css

### PR DESCRIPTION
**Pull request by Polkas**
linked to  https://github.com/insightsengineering/coredev-tasks/issues/225

- remove inline css rule
- DRY
- style at the head - tags$head(includeCSS())

### CODE REVIEW

**Review by nikolas-burkoff — COMMENTED** [review]
A few comments inline - also:
image

File: inst/css/picker_input.css [hunk 1]
```diff
@@ -1,5 +0,0 @@
-
-/* to set the alignment of pickerInput dropdowns */
-.dropdown-menu.open {
```
Comment (nikolas-burkoff) [inline inst/css/picker_input.css:3]: hmm - so the pickerInput is inside optionalselectInput which lives in teal.widgets so not sure it should have been moved to teal?

Comment (Polkas) [reply inst/css/picker_input.css:3]: you are right, I will attach it directly in the optionalselectInput

File: R/verbatim_popup.R [hunk 2]
```diff
@@ -101,31 +101,33 @@ button_click_observer <- function(click_event, copy_button_id, copied_area_id, m
...
+        shiny::modalDialog(
+          shiny::tagList(
+            shiny::tags$div(
+              shiny::actionButton(
+                copy_button_id,
+                "Copy to Clipboard",
+                onclick = paste0("copyToClipboard('", copied_area_id, "')")
+              ),
+              shiny::modalButton("Dismiss"),
+              style = "margin-bottom: 15px;"
```
[the remark this answers is not in the document]
Comment (Polkas) [reply R/verbatim_popup.R]: I leave a 2 inlines as was a separate function and not want to create a separate css files for them. 
I just did it.

[the remark this answers is not in the document]
Comment (nikolas-burkoff) [reply R/verbatim_popup.R]: That's fine just thought it worth pointing out in case it wasn't a conscious decision

**Review by nikolas-burkoff — APPROVED** [review]
👍

**Pull request merged**